supported-smartphones.md

```diff
@@ -97,7 +97,7 @@ The easiest way to verify support is to read the Android Feature flags.
 
 #### I have a phone that isn’t listed and I want to add it.
 
-That’s great! Please follow the methodology to verify its capabilities and create a Pull Request with additional information and screenshots from the tests proving the support. 
+That’s great! Please follow the methodology to verify its capabilities and create a Pull Request with additional information and screenshots from the tests proving the support.
 
 For the Pull Request, please upload the screenshots to the `receiver_proofs/xxx_yyy` folder in the repository where `xxx` is the lowercase smartphone manufacturer (e.g. samsung) and `yyy` is the lowercase smartphone model (e.g. `galaxy_note_10`). Name the screenshots in the folder accordingly to tested capabilities. We recommend to use the following template:
```

Comment:
I see that you changed my intention about using lowercase characters. I am fine with it, but could you please edit this description before we merge it? :)

Reply:
Sorry, I completely missed that. I will update the instructions. It seems more readable to me when you use both lower and upper case letters.

Reply:
I agree that it is more readable. But also a little harder to write correctly which might make it inconsistent in the future. But let's keep it your way now and then we can fine-tune it